### PR TITLE
numworks-epsilon: 15.3.2 -> 15.5.0

### PR DESCRIPTION
**Pull request by erikbackman**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 122760` at fa21cccb run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>numworks-epsilon</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Consider adding a comment explaining the purpose of this patch on the line preceeding.
Near pkgs/applications/science/math/numworks-epsilon/default.nix:41:5:
```
   |
41 |     ./0001-ion-linux-makerules.patch
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/applications/science/math/numworks-epsilon/update.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq common-updater-scripts
+
+latestVersion=$(curl https://api.github.com/repos/numworks/epsilon/tags | jq -r '.[] | .name' | sort --version-sort | tail -1)
+
+update-source-version numworks-epsilon "$latestVersion"
```
Comment (SuperSandro2000) [inline pkgs/applications/science/math/numworks-epsilon/update.sh:6]:
```suggestion
```

File: pkgs/applications/science/math/numworks-epsilon/default.nix [hunk 2]
```diff
@@ -44,10 +52,14 @@ stdenv.mkDerivation rec {
...
+    description = "Simulator for Epsilon, a High-performance graphing calculator operating system";
     homepage = "https://numworks.com/";
     license = licenses.cc-by-nc-sa-40;
     maintainers = with maintainers; [ erikbackman ];
     platforms = [ "x86_64-linux" ];
   };
+
+  passthru = {
+    updateScript = ./update.sh;
+  };
```
Comment (SuperSandro2000) [inline pkgs/applications/science/math/numworks-epsilon/default.nix:64]:
```suggestion
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please use nix-update instead.

**Comment by erikbackman** [comment]
> Please use nix-update instead.

Good to know :+1:

**Pull request merged**